ui/app/src/views/ViewDashboard.tsx

```diff
@@ -22,18 +23,20 @@ const DEFAULT_DASHBOARD_ID = 'node-exporter-full';
  * The View for viewing a Dashboard.
  */
 function ViewDashboard() {
-  const dashboard = useSampleData<DashboardResource>(
-    new URLSearchParams(window.location.search).get('dashboard') || DEFAULT_DASHBOARD_ID
-  );
+  const [query] = useQueryParams({ dashboard: StringParam });
```

Comment:
This API already looks so much like our internal implementation of query string params, I'm wondering where it was a couple years ago. 😂